### ISSUE TITLE
Support Google Home again via Google's new "Local Home SDK"

### ISSUE THREAD

**Issue by SEQUOiA87** [bwssytems/ha-bridge#1200]
Hey! I think it should be possible that HA Bridge supports Google Home / Nest devices again, by using Google's new "Local Home SDK". Here is the official announcement about the SDK being out of beta now: https://developers.googleblog.com/2020/04/local-home-sdk-ready-for-actions.html

Would be great for many users!
What do you think?

Thanks in advance.

**Comment by bwssytems** [comment]
This is cool, but it is for an app on the google home to find local devices. I guess the idea is to create an ha-bridge app that can be added to your google home so it can talk to the bridge.

**Comment by SEQUOiA87** [comment]
Yes, i think that's what would be needed. We would need 2 things here:
1. Easy: Existing local HA Bridge instance to support local fulfillment, i.e. make it possible for Google's Local Home SDK to identify and talk to (controll, query) devices under management of HA Bridge.
2. Hard: An "Actions on Google" public service for HA Bridge of type "Smart Home" (just like Hue, Yeelight and others do offer), including user registry and possibility to know about the user's HA bridge devices (at least a subset of them which the user wants to control via Google Home). Since Google reviews such a service, I think you need to also provide cloud control of the said "public" HA devices as a fallback for cases where the local fulfillment doesn't work...

What do you think?

**Comment by bwssytems** [comment]
The google home sdk should be able to be setup to call the emulated hue interface just fine. So it is making the service.

**Comment by SEQUOiA87** [comment]
> The google home sdk should be able to be setup to call the emulated hue interface just fine. So it is making the service.

Eh, how should that happen? I thought it's not possible anymore to add emulated hue devices to Google Home without an actual Hue hardware setup. As I don't have any Hue devices, I guess I'm lost :)

**Comment by bwssytems** [comment]
If you create a custom service with SDK, it won't know it is hue that it is dealing with...

**Comment by SEQUOiA87** [comment]
> If you create a custom service with SDK, it won't know it is hue that it is dealing with...

Exactly. And that's perfect, because the custom service could even tell Google that my devices are actually no lights but mostly switches. So that would also increase UX.
